Jenkinsfile

```diff
@@ -2,15 +2,19 @@ wrapperConfigurations = [
   Debug: 'dbg',
   Release: ''
 ]
-configuration = 'Release'
+configuration = 'Debug'
 
 xbuildCmd = '/usr/local/bin/xbuild'
-def nuget = '/usr/local/bin/nuget'
+nugetCmd = '/usr/local/bin/nuget'
+def windowsNugetCmd = 'C:\\ProgramData\\chocolatey\\bin\\NuGet.exe'
```

Comment:
Nope, we don't. And that only matters to Jenkins. People building manually (us, included) will just restore packages like they always would - through Visual Studio.